src/parser/bison_parser.y

```diff
@@ -36,8 +36,7 @@
 // Specify code that is included in the generated .h and .c files
 // clang-format off
 %code requires {
-  // clang-format on
```

Comment:
Why remove? Does clang complain about the format?

Reply:
To be honest, gonna have to look into this after that time. Think that clang-format at that time (probably 14 or so) did some weird stuff then.

Reply:
Indeed, clang-format (15) locally screws up the indentation and even adds linebreaks in the bison control lines.